include/libtorrent/buffer.hpp

```diff
@@ -75,7 +68,7 @@ class buffer
 
 		if (size == 0) return;
 
-		size = round_up8(size);
+		size = (size + 7) & (~std::size_t(0x7)); //round_up8(size);
```

Comment:
I think the rounding up does make sense, since the buffer is not resizable. I suppose it mostly makes sense for platforms of ``malloc_size()``. Is there a good reason to collapse the function down to this expression? just fewer lines? I suppose it's not that important, but maybe you could remove the comment with the previous function call